Persona.Merger.Tests/Parser/P5RTblPatcherTests.cs

```diff
@@ -37,4 +38,40 @@ public void PatchTbl_Item_Extend()
         var patched = patcher.Apply(new List<TblPatch>() { patch });
         Assert.Equal(after, patched);
     }
+    
+    [Fact]
+    public void PatchTbl_Generic()
+    {
+        var patches = new List<TblPatch>();
+
+        var original = File.ReadAllBytes(P5RAssets.PDDBefore);
+        var after = File.ReadAllBytes(P5RAssets.PDDAfter);
+        var after2 = File.ReadAllBytes(P5RAssets.PDDAfter2);
+        var after3 = File.ReadAllBytes(P5RAssets.PDDAfter3);
+        var patcher = new GenericPatcher(original);
+        patches.Add(patcher.GeneratePatchGeneric(after, 4));
+        patches.Add(patcher.GeneratePatchGeneric(after2, 4));
+        var patched = patcher.ApplyGeneric(patches);
+
+        Assert.Equal(patched, after3);
+    }
+    
+    [Fact]
+    public void PatchTbl_VISUAL()
+    {
+        var patches = new List<TblPatch>();
+
+        var original = File.ReadAllBytes(P5RAssets.VisualBefore);
+        var after = File.ReadAllBytes(P5RAssets.VisualAfter);
+        var after2 = File.ReadAllBytes(P5RAssets.VisualAfter2);
+        var after3 = File.ReadAllBytes(P5RAssets.VisualAfter3);
```

Comment:
This should be named 'expected', since it's the expected result after applying 2 patches. Same applies to the other test.

Reply:
Done.